ansible/plugins/inventory/nodejs_yaml.py

```diff
@@ -47,7 +47,7 @@
     # providers - validated for consistency
     'provider': ('azure', 'digitalocean', 'ibm', 'joyent', 'linuxonecc',
                  'macstadium', 'marist', 'mininodes', 'msft', 'osuosl',
-                 'rackspace', 'requireio', 'scaleway', 'softlayer', 'voxer',
+                 'orka', 'rackspace', 'requireio', 'scaleway', 'softlayer', 'voxer',
```

Comment:
this is breaking master since the orka hosts were merged into secrets

```
Exception: Failed to parse host: Invalid provider: orka
```

I can fix manually to run Ansible but can we get this merged and have follow-up work later?